### ISSUE TITLE
Progressbar from MiniPlayer is „wrong“

### ISSUE THREAD

**Issue by BKrnk1337** [Automattic/pocket-casts-ios#1873]
### Description

When I am in the Queue view and have the MiniPlayer minimized the progressbar at the top is correct but seems frozen at the bottom

### Step-by-step reproduction instructions

Happens from time to time on the queue page.
When I open the MiniPlayer and minimise it again the progressbad is correct.

### Screenshots or screen recording

![IMG_6561](https://github.com/Automattic/pocket-casts-ios/assets/27677748/dac35129-9430-42b4-b77e-e36e0eaac252)

### Did you search for existing bug reports?

- [X] I have searched for existing bug reports.

### Device, Operating system, and Pocket Casts app version

iPhone 15 Pro Max - iOS 17

**Comment by BKrnk1337** [comment]
I add some more pictures. The behavior is still present.

![image](https://github.com/user-attachments/assets/d9153357-e68e-48d1-b52b-d635bcfe7f1d)

![image](https://github.com/user-attachments/assets/c70a3be2-0b65-4640-8f52-dff3d7539822)

**Comment by rviljoen** [comment]
@BKrnk1337 I tried to replicate this issue, but was not able to do so on the latest version (v7.69). You mention that it happens from time to time. When it happens again, can you please note what pre-ceding actions you took? Seems like this may be an edge-case bug which is triggered under specific conditions.

**Comment by BKrnk1337** [comment]
@rviljoen 

Hello. Thank you for answering. I don't do anything special. Just listing to a podcast with the phone blocked in my pocket. After some podcast (maybe this) the MiniPlayer still shows begin of episode but after opening the player it is nearly ended (or any other progression)

Version is 7.70.00. the bug exist for several versions.

**Here are some Logs - I finished my run at 12:06 and opened the app some minutes after. Maybe you can find something at this timestamp**

```
2024-08-06 10:50:02 Sent position 6802 status 2 for episode Feierabendbier: Gamedesign-No-Gos to server
2024-08-06 10:50:33 saving played up to 6840.53224489796 for episode Feierabendbier: Gamedesign-No-Gos
2024-08-06 10:51:05 saving played up to 6878.932244897959 for episode Feierabendbier: Gamedesign-No-Gos
2024-08-06 10:51:37 saving played up to 6917.332244897959 for episode Feierabendbier: Gamedesign-No-Gos
2024-08-06 10:52:09 saving played up to 6955.732244897959 for episode Feierabendbier: Gamedesign-No-Gos
2024-08-06 10:52:10 Sent position 6955 status 2 for episode Feierabendbier: Gamedesign-No-Gos to server
2024-08-06 10:52:41 saving played up to 6994.132244897959 for episode Feierabendbier: Gamedesign-No-Gos
2024-08-06 10:53:13 saving played up to 7032.53224489796 for episode Feierabendbier: Gamedesign-No-Gos
2024-08-06 10:53:45 saving played up to 7070.932244897959 for episode Feierabendbier: Gamedesign-No-Gos
2024-08-06 10:54:17 saving played up to 7109.436734693878 for episode Feierabendbier: Gamedesign-No-Gos
2024-08-06 10:54:18 Sent position 7109 status 2 for episode Feierabendbier: Gamedesign-No-Gos to server
2024-08-06 10:54:49 saving played up to 7147.888979591837 for episode Feierabendbier: Gamedesign-No-Gos
2024-08-06 10:55:21 saving played up to 7186.288979591836 for episode Feierabendbier: Gamedesign-No-Gos
2024-08-06 10:55:53 saving played up to 7224.871836734694 for episode Feierabendbier: Gamedesign-No-Gos
2024-08-06 10:56:25 saving played up to 7263.324081632653 for episode Feierabendbier: Gamedesign-No-Gos
2024-08-06 10:56:26 Sent position 7263 status 2 for episode Feierabendbier: Gamedesign-No-Gos to server
2024-08-06 10:56:57 saving played up to 7301.697959183673 for episode Feierabendbier: Gamedesign-No-Gos
2024-08-06 10:57:29 saving played up to 7340.097959183673 for episode Feierabendbier: Gamedesign-No-Gos
2024-08-06 10:58:01 saving played up to 7378.70693877551 for episode Feierabendbier: Gamedesign-No-Gos
2024-08-06 10:58:33 saving played up to 7417.106938775511 for episode Feierabendbier: Gamedesign-No-Gos
2024-08-06 10:58:34 Sent position 7417 status 2 for episode Feierabendbier: Gamedesign-No-Gos to server
2024-08-06 10:59:05 saving played up to 7455.50693877551 for episode Feierabendbier: Gamedesign-No-Gos
2024-08-06 10:59:37 saving played up to 7494.142040816327 for episode Feierabendbier: Gamedesign-No-Gos
2024-08-06 11:00:08 Remote control: pauseCommand
2024-08-06 11:00:08 pausing playback
2024-08-06 11:00:08 saving played up to 7530.9224489795915 for episode Feierabendbier: Gamedesign-No-Gos
2024-08-06 11:00:08 cleanupCurrentPlayer permanent? true
2024-08-06 11:00:11 deactiveAudioSession succeeded
2024-08-06 11:00:13 deactiveAudioSession succeeded
2024-08-06 11:00:34 Remote control: playCommand, treating as playPause
2024-08-06 11:00:34 Loading Feierabendbier: Gamedesign-No-Gos with UUID 9ca67a59-e672-46a0-ac17-1393c93faab7 autoPlay false
2024-08-06 11:00:34 saving played up to 7530.948571428571 for episode Feierabendbier: Gamedesign-No-Gos
2024-08-06 11:00:34 cleanupCurrentPlayer permanent? false
2024-08-06 11:00:34 Using EffectsPlayer
2024-08-06 11:00:34 activating audio session succeeded
2024-08-06 11:00:34 Not enough time passed since this episode was last paused, no time adjustment required
2024-08-06 11:00:36 saving played up to 7533.090612244898 for episode Feierabendbier: Gamedesign-No-Gos
2024-08-06 11:00:37 Sent position 7533 status 2 for episode Feierabendbier: Gamedesign-No-Gos to server
2024-08-06 11:01:08 saving played up to 7571.568979591837 for episode Feierabendbier: Gamedesign-No-Gos
2024-08-06 11:01:40 saving played up to 7609.995102040816 for episode Feierabendbier: Gamedesign-No-Gos
2024-08-06 11:01:42 EffectsPlayer got to end of episode, calling finished playing
2024-08-06 11:01:42 Autoplay: reset the last playlist
2024-08-06 11:01:42 Finished playing Feierabendbier: Gamedesign-No-Gos
2024-08-06 11:01:42 Sending playback completed to API server
2024-08-06 11:01:42 Play Next Episode Altbier: Age of Mythology
2024-08-06 11:01:42 Remove Top Episode Feierabendbier: Gamedesign-No-Gos
2024-08-06 11:01:42 PlaybackQueue: removing Feierabendbier: Gamedesign-No-Gos episode
2024-08-06 11:01:42 cleanupCurrentPlayer permanent? false
2024-08-06 11:01:42 Loading Altbier: Age of Mythology with UUID 015008bf-8515-4375-bcf8-69ccaedc8fac autoPlay false
2024-08-06 11:01:42 Show Info: returning cached data for episode 9ca67a59-e672-46a0-ac17-1393c93faab7
2024-08-06 11:01:42 cleanupCurrentPlayer permanent? false
2024-08-06 11:01:42 Using EffectsPlayer
2024-08-06 11:01:42 activating audio session succeeded
2024-08-06 11:01:42 Sent position 7610 status 3 for episode Feierabendbier: Gamedesign-No-Gos to server
2024-08-06 11:01:42 Show Info: returning cached data for episode 015008bf-8515-4375-bcf8-69ccaedc8fac
2024-08-06 11:01:43 Show Info: returning cached data for episode 015008bf-8515-4375-bcf8-69ccaedc8fac
2024-08-06 11:01:43 ChapterManager: using file chapters
2024-08-06 11:01:47 RetrieveCustomFilesTask - not modified, no changes required
2024-08-06 11:01:47 UpNextSyncTask [createUpNextSyncRequest]: Sync Reason? None
2024-08-06 11:01:47 UpNextSyncTask: Syncing Up Next, sending 1 changes, modified time 1722933022802
2024-08-06 11:01:47 UpNextSyncTask: server copy matches our copy, no action required
2024-08-06 11:02:13 saving played up to 37.06775510204081 for episode Altbier: Age of Mythology
2024-08-06 11:02:45 saving played up to 75.4938775510204 for episode Altbier: Age of Mythology
2024-08-06 11:02:45 Sent position 75 status 2 for episode Altbier: Age of Mythology to server
2024-08-06 11:03:17 saving played up to 113.86775510204082 for episode Altbier: Age of Mythology
2024-08-06 11:03:49 saving played up to 152.45061224489797 for episode Altbier: Age of Mythology
2024-08-06 11:04:21 saving played up to 190.85061224489795 for episode Altbier: Age of Mythology
2024-08-06 11:04:53 saving played up to 229.61632653061224 for episode Altbier: Age of Mythology
2024-08-06 11:04:53 Sent position 229 status 2 for episode Altbier: Age of Mythology to server
2024-08-06 11:05:25 saving played up to 268.0163265306122 for episode Altbier: Age of Mythology
2024-08-06 11:05:57 saving played up to 306.41632653061225 for episode Altbier: Age of Mythology
2024-08-06 11:06:29 saving played up to 344.86857142857144 for episode Altbier: Age of Mythology
2024-08-06 11:07:01 saving played up to 383.2685714285714 for episode Altbier: Age of Mythology
2024-08-06 11:07:01 Sent position 383 status 2 for episode Altbier: Age of Mythology to server
2024-08-06 11:07:33 saving played up to 421.66857142857145 for episode Altbier: Age of Mythology
2024-08-06 11:08:05 saving played up to 460.06857142857143 for episode Altbier: Age of Mythology
2024-08-06 11:08:37 saving played up to 498.4685714285714 for episode Altbier: Age of Mythology
2024-08-06 11:09:09 saving played up to 537.025306122449 for episode Altbier: Age of Mythology
2024-08-06 11:09:09 Sent position 537 status 2 for episode Altbier: Age of Mythology to server
2024-08-06 11:09:41 saving played up to 575.425306122449 for episode Altbier: Age of Mythology
2024-08-06 11:10:13 saving played up to 613.8775510204082 for episode Altbier: Age of Mythology
2024-08-06 11:10:45 saving played up to 652.2775510204082 for episode Altbier: Age of Mythology
2024-08-06 11:11:17 saving played up to 690.6775510204081 for episode Altbier: Age of Mythology
2024-08-06 11:11:17 Sent position 690 status 2 for episode Altbier: Age of Mythology to server
2024-08-06 11:11:49 saving played up to 729.0775510204081 for episode Altbier: Age of Mythology
2024-08-06 11:12:21 saving played up to 767.4775510204082 for episode Altbier: Age of Mythology
2024-08-06 11:12:53 saving played up to 806.138775510204 for episode Altbier: Age of Mythology
2024-08-06 11:13:25 saving played up to 844.5387755102041 for episode Altbier: Age of Mythology
2024-08-06 11:13:25 Sent position 844 status 2 for episode Altbier: Age of Mythology to server
2024-08-06 11:13:57 saving played up to 882.9648979591836 for episode Altbier: Age of Mythology
2024-08-06 11:14:29 saving played up to 921.3387755102041 for episode Altbier: Age of Mythology
2024-08-06 11:15:01 saving played up to 959.738775510204 for episode Altbier: Age of Mythology
2024-08-06 11:15:33 saving played up to 998.138775510204 for episode Altbier: Age of Mythology
2024-08-06 11:15:33 Sent position 998 status 2 for episode Altbier: Age of Mythology to server
2024-08-06 11:16:05 saving played up to 1036.669387755102 for episode Altbier: Age of Mythology
2024-08-06 11:16:37 saving played up to 1075.2522448979591 for episode Altbier: Age of Mythology
2024-08-06 11:17:09 saving played up to 1113.6522448979592 for episode Altbier: Age of Mythology
2024-08-06 11:17:41 saving played up to 1152.052244897959 for episode Altbier: Age of Mythology
2024-08-06 11:17:41 Sent position 1152 status 2 for episode Altbier: Age of Mythology to server
2024-08-06 11:18:13 saving played up to 1190.4783673469387 for episode Altbier: Age of Mythology
2024-08-06 11:18:45 saving played up to 1228.8522448979593 for episode Altbier: Age of Mythology
2024-08-06 11:19:17 saving played up to 1267.2522448979591 for episode Altbier: Age of Mythology
2024-08-06 11:19:49 saving played up to 1305.6522448979592 for episode Altbier: Age of Mythology
2024-08-06 11:19:49 Sent position 1305 status 2 for episode Altbier: Age of Mythology to server
2024-08-06 11:20:21 saving played up to 1344.0783673469389 for episode Altbier: Age of Mythology
2024-08-06 11:20:53 saving played up to 1382.4522448979592 for episode Altbier: Age of Mythology
2024-08-06 11:21:25 saving played up to 1420.8522448979593 for episode Altbier: Age of Mythology
2024-08-06 11:21:57 saving played up to 1459.4351020408162 for episode Altbier: Age of Mythology
2024-08-06 11:21:57 Sent position 1459 status 2 for episode Altbier: Age of Mythology to server
2024-08-06 11:22:29 saving played up to 1497.8612244897959 for episode Altbier: Age of Mythology
2024-08-06 11:23:01 saving played up to 1536.2351020408164 for episode Altbier: Age of Mythology
2024-08-06 11:23:33 saving played up to 1574.6351020408163 for episode Altbier: Age of Mythology
2024-08-06 11:24:05 saving played up to 1613.061224489796 for episode Altbier: Age of Mythology
2024-08-06 11:24:05 Sent position 1613 status 2 for episode Altbier: Age of Mythology to server
2024-08-06 11:24:37 saving played up to 1651.461224489796 for episode Altbier: Age of Mythology
2024-08-06 11:25:09 saving played up to 1689.8351020408163 for episode Altbier: Age of Mythology
2024-08-06 11:25:41 saving played up to 1728.2351020408164 for episode Altbier: Age of Mythology
2024-08-06 11:26:13 saving played up to 1766.6612244897958 for episode Altbier: Age of Mythology
2024-08-06 11:26:13 Sent position 1766 status 2 for episode Altbier: Age of Mythology to server
2024-08-06 11:26:45 saving played up to 1805.061224489796 for episode Altbier: Age of Mythology
2024-08-06 11:27:17 saving played up to 1843.4351020408162 for episode Altbier: Age of Mythology
2024-08-06 11:27:49 saving played up to 1882.7755102040817 for episode Altbier: Age of Mythology
2024-08-06 11:28:21 saving played up to 1921.2016326530613 for episode Altbier: Age of Mythology
2024-08-06 11:28:21 Sent position 1921 status 2 for episode Altbier: Age of Mythology to server
2024-08-06 11:28:53 saving played up to 1959.6016326530612 for episode Altbier: Age of Mythology
2024-08-06 11:29:25 saving played up to 1997.9755102040817 for episode Altbier: Age of Mythology
2024-08-06 11:29:57 saving played up to 2036.3755102040816 for episode Altbier: Age of Mythology
2024-08-06 11:30:29 saving played up to 2074.801632653061 for episode Altbier: Age of Mythology
2024-08-06 11:30:29 Sent position 2074 status 2 for episode Altbier: Age of Mythology to server
2024-08-06 11:31:01 saving played up to 2113.4367346938775 for episode Altbier: Age of Mythology
2024-08-06 11:31:33 saving played up to 2151.810612244898 for episode Altbier: Age of Mythology
2024-08-06 11:32:05 saving played up to 2190.210612244898 for episode Altbier: Age of Mythology
2024-08-06 11:32:37 saving played up to 2228.6367346938778 for episode Altbier: Age of Mythology
2024-08-06 11:32:37 Sent position 2228 status 2 for episode Altbier: Age of Mythology to server
2024-08-06 11:33:09 saving played up to 2267.4808163265307 for episode Altbier: Age of Mythology
2024-08-06 11:33:41 saving played up to 2305.854693877551 for episode Altbier: Age of Mythology
2024-08-06 11:34:13 saving played up to 2344.385306122449 for episode Altbier: Age of Mythology
2024-08-06 11:34:45 saving played up to 2382.785306122449 for episode Altbier: Age of Mythology
2024-08-06 11:34:45 Sent position 2382 status 2 for episode Altbier: Age of Mythology to server
2024-08-06 11:35:17 saving played up to 2421.1853061224488 for episode Altbier: Age of Mythology
2024-08-06 11:35:49 saving played up to 2459.585306122449 for episode Altbier: Age of Mythology
2024-08-06 11:36:21 saving played up to 2497.985306122449 for episode Altbier: Age of Mythology
2024-08-06 11:36:53 saving played up to 2536.385306122449 for episode Altbier: Age of Mythology
2024-08-06 11:36:53 Sent position 2536 status 2 for episode Altbier: Age of Mythology to server
2024-08-06 11:37:25 saving played up to 2575.229387755102 for episode Altbier: Age of Mythology
2024-08-06 11:37:57 saving played up to 2613.6032653061225 for episode Altbier: Age of Mythology
2024-08-06 11:38:29 saving played up to 2652.0032653061226 for episode Altbier: Age of Mythology
2024-08-06 11:39:01 saving played up to 2690.429387755102 for episode Altbier: Age of Mythology
2024-08-06 11:39:01 Sent position 2690 status 2 for episode Altbier: Age of Mythology to server
2024-08-06 11:39:33 saving played up to 2728.829387755102 for episode Altbier: Age of Mythology
2024-08-06 11:40:05 saving played up to 2767.6473469387756 for episode Altbier: Age of Mythology
2024-08-06 11:40:37 saving played up to 2806.0473469387757 for episode Altbier: Age of Mythology
2024-08-06 11:41:09 saving played up to 2844.473469387755 for episode Altbier: Age of Mythology
2024-08-06 11:41:09 Sent position 2844 status 2 for episode Altbier: Age of Mythology to server
2024-08-06 11:41:41 saving played up to 2882.8473469387754 for episode Altbier: Age of Mythology
2024-08-06 11:42:13 saving played up to 2921.2473469387755 for episode Altbier: Age of Mythology
2024-08-06 11:42:45 saving played up to 2959.6473469387756 for episode Altbier: Age of Mythology
2024-08-06 11:43:17 saving played up to 2998.0734693877553 for episode Altbier: Age of Mythology
2024-08-06 11:43:17 Sent position 2998 status 2 for episode Altbier: Age of Mythology to server
2024-08-06 11:43:49 saving played up to 3036.473469387755 for episode Altbier: Age of Mythology
2024-08-06 11:44:21 saving played up to 3074.8473469387754 for episode Altbier: Age of Mythology
2024-08-06 11:44:53 saving played up to 3113.325714285714 for episode Altbier: Age of Mythology
2024-08-06 11:45:25 saving played up to 3151.7257142857143 for episode Altbier: Age of Mythology
2024-08-06 11:45:25 Sent position 3151 status 2 for episode Altbier: Age of Mythology to server
2024-08-06 11:45:57 saving played up to 3190.151836734694 for episode Altbier: Age of Mythology
2024-08-06 11:46:29 saving played up to 3228.5257142857145 for episode Altbier: Age of Mythology
2024-08-06 11:47:01 saving played up to 3266.925714285714 for episode Altbier: Age of Mythology
2024-08-06 11:47:33 saving played up to 3305.325714285714 for episode Altbier: Age of Mythology
2024-08-06 11:47:33 Sent position 3305 status 2 for episode Altbier: Age of Mythology to server
2024-08-06 11:48:05 saving played up to 3343.751836734694 for episode Altbier: Age of Mythology
2024-08-06 11:48:37 saving played up to 3382.1257142857144 for episode Altbier: Age of Mythology
2024-08-06 11:49:09 saving played up to 3420.5257142857145 for episode Altbier: Age of Mythology
2024-08-06 11:49:41 saving played up to 3458.951836734694 for episode Altbier: Age of Mythology
2024-08-06 11:49:41 Sent position 3458 status 2 for episode Altbier: Age of Mythology to server
2024-08-06 11:50:13 saving played up to 3497.534693877551 for episode Altbier: Age of Mythology
2024-08-06 11:50:45 saving played up to 3535.9085714285716 for episode Altbier: Age of Mythology
2024-08-06 11:51:17 saving played up to 3574.308571428571 for episode Altbier: Age of Mythology
2024-08-06 11:51:49 saving played up to 3612.734693877551 for episode Altbier: Age of Mythology
2024-08-06 11:51:49 Sent position 3612 status 2 for episode Altbier: Age of Mythology to server
2024-08-06 11:52:21 saving played up to 3651.134693877551 for episode Altbier: Age of Mythology
2024-08-06 11:52:53 saving played up to 3689.534693877551 for episode Altbier: Age of Mythology
2024-08-06 11:53:25 saving played up to 3727.934693877551 for episode Altbier: Age of Mythology
2024-08-06 11:53:57 saving played up to 3766.3608163265308 for episode Altbier: Age of Mythology
2024-08-06 11:53:57 Sent position 3766 status 2 for episode Altbier: Age of Mythology to server
2024-08-06 11:54:29 saving played up to 3804.7608163265304 for episode Altbier: Age of Mythology
2024-08-06 11:55:01 saving played up to 3843.134693877551 for episode Altbier: Age of Mythology
2024-08-06 11:55:33 saving played up to 3881.743673469388 for episode Altbier: Age of Mythology
2024-08-06 11:56:05 saving played up to 3920.1697959183675 for episode Altbier: Age of Mythology
2024-08-06 11:56:05 Sent position 3920 status 2 for episode Altbier: Age of Mythology to server
2024-08-06 11:56:37 saving played up to 3958.569795918367 for episode Altbier: Age of Mythology
2024-08-06 11:57:09 saving played up to 3996.9436734693877 for episode Altbier: Age of Mythology
2024-08-06 11:57:41 saving played up to 4035.3697959183673 for episode Altbier: Age of Mythology
2024-08-06 11:58:13 saving played up to 4073.795918367347 for episode Altbier: Age of Mythology
2024-08-06 11:58:13 Sent position 4073 status 2 for episode Altbier: Age of Mythology to server
2024-08-06 11:58:45 saving played up to 4112.66612244898 for episode Altbier: Age of Mythology
2024-08-06 11:59:17 saving played up to 4151.04 for episode Altbier: Age of Mythology
2024-08-06 11:59:49 saving played up to 4189.44 for episode Altbier: Age of Mythology
2024-08-06 12:00:21 saving played up to 4228.702040816326 for episode Altbier: Age of Mythology
2024-08-06 12:00:21 Sent position 4228 status 2 for episode Altbier: Age of Mythology to server
2024-08-06 12:00:53 saving played up to 4267.755102040816 for episode Altbier: Age of Mythology
2024-08-06 12:01:25 saving played up to 4306.1289795918365 for episode Altbier: Age of Mythology
2024-08-06 12:01:57 saving played up to 4344.555102040817 for episode Altbier: Age of Mythology
2024-08-06 12:02:29 saving played up to 4382.981224489796 for episode Altbier: Age of Mythology
2024-08-06 12:02:29 Sent position 4382 status 2 for episode Altbier: Age of Mythology to server
2024-08-06 12:03:01 saving played up to 4421.381224489796 for episode Altbier: Age of Mythology
2024-08-06 12:03:33 saving played up to 4459.755102040816 for episode Altbier: Age of Mythology
2024-08-06 12:04:05 saving played up to 4498.155102040816 for episode Altbier: Age of Mythology
2024-08-06 12:04:37 saving played up to 4536.581224489796 for episode Altbier: Age of Mythology
2024-08-06 12:04:37 Sent position 4536 status 2 for episode Altbier: Age of Mythology to server
2024-08-06 12:05:09 saving played up to 4575.1640816326535 for episode Altbier: Age of Mythology
2024-08-06 12:05:41 saving played up to 4613.72081632653 for episode Altbier: Age of Mythology
2024-08-06 12:06:13 saving played up to 4652.120816326531 for episode Altbier: Age of Mythology
2024-08-06 12:06:45 saving played up to 4690.729795918367 for episode Altbier: Age of Mythology
2024-08-06 12:06:45 Sent position 4690 status 2 for episode Altbier: Age of Mythology to server
2024-08-06 12:06:46 Remote control: pauseCommand
2024-08-06 12:06:46 pausing playback
2024-08-06 12:06:46 saving played up to 4692.819591836735 for episode Altbier: Age of Mythology
2024-08-06 12:06:46 cleanupCurrentPlayer permanent? true
2024-08-06 12:06:50 deactiveAudioSession succeeded
2024-08-06 12:06:52 deactiveAudioSession succeeded
2024-08-06 12:09:40 Show Info: returning cached data for episode 015008bf-8515-4375-bcf8-69ccaedc8fac
2024-08-06 12:09:44 Refresh complete found 2 new episodes
2024-08-06 12:09:45 RetrieveCustomFilesTask - not modified, no changes required
2024-08-06 12:09:45 UpNextSyncTask [createUpNextSyncRequest]: Sync Reason? None
2024-08-06 12:09:45 UpNextSyncTask: Syncing Up Next, sending 0 changes, modified time 1722934907465
2024-08-06 12:09:45 UpNextSyncTask: Server returned not modified to Up Next sync, no changes required
2024-08-06 12:09:45 [Folders] SyncTask: Number of changed episodes: 3
2024-08-06 12:09:45 [Folders] SyncTask: sending stats changes
2024-08-06 12:09:45 SyncTask: sending 4 changed items to the server
2024-08-06 12:09:46 SyncTask: Found 0 bookmarks to import
2024-08-06 12:10:11 Received subscription status paid : 1, platform : 3, frequency : 2, giftDays : 0, expiryDate : 2025-07-15 00:00:00 +0000
2024-08-06 12:10:11 Sync succeeded
2024-08-06 12:10:14 Show Info: requesting info for podcast a6ddfa20-be20-0130-2e56-723c91aeae46
2024-08-06 12:10:14 Show Info: requesting info for podcast 96a84120-2597-012e-056b-00163e1b201c
2024-08-06 12:10:14 Loading Altbier: Age of Mythology with UUID 015008bf-8515-4375-bcf8-69ccaedc8fac autoPlay false
2024-08-06 12:10:14 saving played up to 4692.845714285714 for episode Altbier: Age of Mythology
2024-08-06 12:10:14 cleanupCurrentPlayer permanent? false
2024-08-06 12:10:14 Using EffectsPlayer
2024-08-06 12:10:15 Methodisch inkorrekt! updated from cache server
2024-08-06 12:10:15 Downloading episode Mi309 – "Millisiebert im Technikmuseum", autoDownloadStatus: autoDownloaded, previousDownloadFailed: false
2024-08-06 12:10:15 SWR Kultur Forum updated from cache server
2024-08-06 12:10:15 Downloading episode Protz, Pracht und Prunk – Wie viel Luxus braucht der Mensch?, autoDownloadStatus: autoDownloaded, previousDownloadFailed: false
2024-08-06 12:10:15 Show Info: request succeeded for podcast 96a84120-2597-012e-056b-00163e1b201c.
2024-08-06 12:10:15 Show Info: request succeeded for podcast a6ddfa20-be20-0130-2e56-723c91aeae46.
2024-08-06 12:38:24 Refresh failed: with error The request timed out., status code 0
2024-08-06 12:38:24 Podcast refresh failed with message: none. See previous log for more details.
2024-08-06 12:38:28 Refresh complete found 0 new episodes
2024-08-06 12:38:28 RetrieveCustomFilesTask - not modified, no changes required
2024-08-06 12:38:28 UpNextSyncTask [createUpNextSyncRequest]: Sync Reason? None
2024-08-06 12:38:28 UpNextSyncTask: Syncing Up Next, sending 0 changes, modified time 1722934907465
2024-08-06 12:38:28 UpNextSyncTask: Server returned not modified to Up Next sync, no changes required
2024-08-06 12:38:28 [Folders] SyncTask: Number of changed episodes: 2
2024-08-06 12:38:28 [Folders] SyncTask: sending stats changes
2024-08-06 12:38:28 SyncTask: sending 3 changed items to the server
2024-08-06 12:38:28 SyncTask: Found 0 bookmarks to import
2024-08-06 12:38:30 Received subscription status paid : 1, platform : 3, frequency : 2, giftDays : 0, expiryDate : 2025-07-15 00:00:00 +0000
2024-08-06 12:38:30 Sync succeeded

```

**Comment by BKrnk1337** [comment]
@rviljoen Can I send you a screen recording via mail?

**Comment by rviljoen** [comment]
> @rviljoen Can I send you a screen recording via mail?

Hi @BKrnk1337 - you can add the recording here if you like so that the entire team can see it. I'm just a contributor and don't work for Pocket Casts. If the recording is sensitive then you can always mail it to Pocket Casts support. 

I have debugged this one a bit without success, but will look at it again over the weekend. There is some logic in the code to not update mini player progress if the full player is visible, which makes sense. My gut tells me there is a bug somewhere where the state is not update correctly, i.e. mini player is visible but it thinks that full player is still active, even when it is not. Just my theory at this point 😀

**Comment by BKrnk1337** [comment]
https://github.com/user-attachments/assets/91615924-1919-490f-84cb-73e669889d1f

**Comment by rviljoen** [comment]
Had a look at this again and cannot find the logic bug in the code, nor can I reproduce it. @BKrnk1337 If you find a way to consistently reproduce this then please let me know. I'll keep looking to see if I can spot it on my side.

**Comment by BKrnk1337** [comment]
@rviljoen 

Just now it happened again. I did not do anything in particular special. Just listen to a podcast and leave my house (switch from WiFi to 5G), then the episode ends and the next in the queue starts. 

I was using the app during this but not opening the player just digging around on the discovery page or in the podcast settings (submitted a request to the engineers)

Now when I check back I see that the MiniPlayer shows a indicator at the end but clearly this podcast have more then 90 minutes more runtime.

Maybe you can take a look at the fact that a episode is finished and the next one starts.

**Comment by rviljoen** [comment]
@BKrnk1337 Thanks for the extra info. I did previously test the use case of the episode finishing and new one starting while app was in background, but still could not reproduce. I think the only potential difference is that, to save time, I put the app in background 20s before end of episode so that I did not have to wait that long for the episode change as part of the test. What I will do now is to pay closer attention to the mini player as part of my normal listen routine, which sounds very similar to what you described above. What I will do is to add more logging to my local debug build and see if I can find any clues.
